### ISSUE TITLE
Add HTTP(S) proxy support in the build process

### ISSUE THREAD

**Issue by trexmaster** [MLstate/PEPS#6]

**Comment by hbbio** [comment]
Can you submit a pull request?

**Comment by trexmaster** [comment]
I'm not used to git so I'll answer with maybe :) I've forked the project, so I'll try to work there and when I'm confident enough I can make a pull request without breaking stuff I'll do it.

As a side note, I just found out that Docker is in the process of adding build-time environment variables (see https://github.com/docker/docker/pull/9176) which would probably solve this issue.

**Comment by hbbio** [comment]
Yes, a few new Docker features would definitely help! Also, we reviewed wether the added complexity or running `etcd` in containers would be a better replacement for all configuration currently in `/etc/peps` but decided to wait until things stabilize a bit on the Docker side.

And we will review pull requests anyway.
